### PR TITLE
feat(suite-web-landing): Add OG meta tags

### LINKED ISSUE #2673: Add OG meta tags 🚀
**Is your feature request related to a problem? Please describe.**
Website looks unfinished without meta tags 😅 

**Describe the solution you'd like**
Add them!

**Describe alternatives you've considered**
Add them later!

**Additional context**
Selection_937

### PR DESCRIPTION
**Pull request by tomexx**
- added social meta tags

### CODE REVIEW

File: packages/suite-web-landing/pages/_document.tsx [hunk 1]
```diff
@@ -39,6 +39,38 @@ export default class MyDocument extends Document {
...
                 <Head>
+                    <meta name="title" content="Trezor Suite" />
+                    <meta
+                        name="description"
+                        content="New desktop &amp; browser app for Trezor hardware wallets. Trezor Suite brings big improvements across our three key pillars of usability, security and privacy."
+                    />
+
+                    {/* Open Graph / Facebook */}
+                    <meta property="og:type" content="website" />
+                    <meta property="og:url" content="https://metatags.io/" />
```
Comment (prusnak) [inline packages/suite-web-landing/pages/_document.tsx]:
```suggestion
                    <meta property="og:url" content="https://suite.trezor.io/" />
```

File: packages/suite-web-landing/pages/_document.tsx [hunk 2]
```diff
@@ -39,6 +39,38 @@ export default class MyDocument extends Document {
...
+                        content="New desktop &amp; browser app for Trezor hardware wallets. Trezor Suite brings big improvements across our three key pillars of usability, security and privacy."
+                    />
+                    <meta
+                        property="og:image"
+                        content="https://metatags.io/assets/meta-tags-16a33a6a8531e519cc0936fbba0ad904e52d35f34a46c97a2c9f6f7dd7d336f2.png"
+                    />
+
+                    {/* Twitter */}
+                    <meta property="twitter:card" content="summary_large_image" />
+                    <meta property="twitter:url" content="https://metatags.io/" />
```
Comment (prusnak) [inline packages/suite-web-landing/pages/_document.tsx]:
```suggestion
                    <meta property="twitter:url" content="https://suite.trezor.io/" />
```

File: packages/suite-web-landing/pages/_document.tsx [hunk 3]
```diff
@@ -39,6 +39,38 @@ export default class MyDocument extends Document {
...
+                    <meta property="og:type" content="website" />
+                    <meta property="og:url" content="https://metatags.io/" />
+                    <meta property="og:title" content="Trezor Suite" />
+                    <meta
+                        property="og:description"
+                        content="New desktop &amp; browser app for Trezor hardware wallets. Trezor Suite brings big improvements across our three key pillars of usability, security and privacy."
+                    />
+                    <meta
+                        property="og:image"
+                        content="https://metatags.io/assets/meta-tags-16a33a6a8531e519cc0936fbba0ad904e52d35f34a46c97a2c9f6f7dd7d336f2.png"
```
Comment (prusnak) [inline packages/suite-web-landing/pages/_document.tsx]: replace with local copy of https://user-images.githubusercontent.com/29627086/97498925-4eda6f80-196d-11eb-8f54-ae8cfe76ce62.png

File: packages/suite-web-landing/pages/_document.tsx [hunk 4]
```diff
@@ -39,6 +39,38 @@ export default class MyDocument extends Document {
...
+                    <meta property="twitter:card" content="summary_large_image" />
+                    <meta property="twitter:url" content="https://metatags.io/" />
+                    <meta property="twitter:title" content="Trezor Suite" />
+                    <meta
+                        property="twitter:description"
+                        content="New desktop &amp; browser app for Trezor hardware wallets. Trezor Suite brings major improvements across our three key pillars of usability, security and privacy."
+                    />
+                    <meta
+                        property="twitter:image"
+                        content="https://metatags.io/assets/meta-tags-16a33a6a8531e519cc0936fbba0ad904e52d35f34a46c97a2c9f6f7dd7d336f2.png"
```
Comment (prusnak) [inline packages/suite-web-landing/pages/_document.tsx]: replace with local copy of https://user-images.githubusercontent.com/29627086/97498925-4eda6f80-196d-11eb-8f54-ae8cfe76ce62.png

**Review by prusnak — CHANGES_REQUESTED** [review]

**Review by prusnak — APPROVED** [review]

**Review by keraf — APPROVED** [review]

**Pull request merged**